### PR TITLE
[Project] Enable cfg import optimize and quantize

### PR DESCRIPTION
**Pull request by seanshpark**
This will eable cfg import to add optimize and quantize jobs.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Review by llFreetimell — APPROVED** [review]

**Comment by seanshpark** [comment]
From draft #70

File: src/Project/BuilderCfgFile.ts [hunk 1]
```diff
@@ -78,6 +82,32 @@ export class BuilderCfgFile extends EventEmitter implements helpers.FileSelector
...
+    console.log('optimize = ', optimize);
+    this.jobOwner.addJob(optimize);
+    this.logger.outputLine('Add Optimize: ' + inputModel);
+  }
+
+  private cfgQuantize(prop: any) {
+    let quantize = new JobQuantize();
+    quantize.inputPath = prop[K_INPUT_PATH];
+    quantize.outputPath = prop[K_OUTPUT_PATH];
+
```
Comment (mhs4670go) [inline src/Project/BuilderCfgFile.ts:102]: @seanshpark Other options will be added?

Comment (seanshpark) [reply src/Project/BuilderCfgFile.ts:102]: Yup, shortly after current draft all lands :)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**